src/components/Products.js

```diff
@@ -13,13 +13,19 @@ const Products = ({products}) => {
         height: '200px'
       };
 
-    const listItems = products.map((product) =>
-        <div key={product.id} style={itemsStyle}>
-            <p>{product.data.name}</p>
-            <p>{product.tags.join(" - ")}</p>
-            <img src={product.data.mainimage.url} alt={product.data.mainimage.alt} style={imageStyle} /> <br/>
-            <p>${product.data.price}</p>
-        </div>
+    const listItems = products.map((product) => {
+        if (!filters || filters.length === 0 || filters.includes(product.data.category.id)) {
```

Comment:
This should just be `filters.includes(product.data.category.id)` or am i missing something ?